content/guides/java/tracers.md

```diff
@@ -2,7 +2,67 @@
 title: "Java: Tracers"
 ---
 
-* Setting up your tracer
-* Using the Global Tracer
-* Accessing the Active Span
-* Accessing tracer specific features.
\ No newline at end of file
+#### Setting up your Tracer
+
+Different **Tracer** implementations vary in how and what parameters they receive at initialization time, such as:
+
+* Component name for this application's traces.
+* Tracing endpoint.
+* Tracing credentials.
+* Sampling strategy.
+
+
+For example, initializing the **Tracer** implementation of `Jaeger` looks like this:
+
+```java
+  import io.opentracing.Tracer;
+
+  SamplerConfiguration samplerConfig = new SamplerConfiguration("const", 1);
+  ReporterConfiguration reporterConfig = new ReporterConfiguration(true, null, null, null, null);
+  Configuration config = new Configuration(service, samplerConfig, reporterConfig);
+
+  // Get the actual OpenTracing-compatible Tracer.
+  Tracer tracer = config.getTracer();
+```
```

Comment:
It would be worth mentioning the Tracer Resolver as well. Something like:

> If your tracer supports it, the [`TracerResolver`](https://github.com/opentracing-contrib/java-tracerresolver) can also be used. With this approach, there's no tracer-specific initialization code in your application: `Tracer tracer = TracerResolver.resolveTracer();`

Reply:
Oh, yes, had forgotten about it ;)